### PR TITLE
Add 2.1.3 > 3 minimum

### PR DESCRIPTION
**Pull request by boblangley**

### CODE REVIEW

File: src/ServiceControlInstaller.Engine/Configuration/ServiceControl/UpgradeControl.Details.cs [hunk 1]
```diff
@@ -11,7 +11,13 @@ public partial class UpgradeControl
                 RecommendedUpgradeVersion = new Version(1, 48, 0),
                 DataBaseUpdate = true,
                 DeleteIndexes = true
+            },
+            new UpgradeInfo(new Version(3, 0), new Version(2, 1, 3)) //https://github.com/Particular/ServiceControl/issues/1228
+            {
+                RecommendedUpgradeVersion = new Version(2, 1, 3),
+                DataBaseUpdate = false,
+                DeleteIndexes = false
```
[the remark this answers is not in the document]
Comment (danielmarbach) [reply src/ServiceControlInstaller.Engine/Configuration/ServiceControl/UpgradeControl.Details.cs]: Indexes should not be deleted. We have not touched the storage format

**Comment by mikeminutillo** [comment]
What is the reasoning for setting this minimum?

**Comment by danielmarbach** [comment]
We agreed to train the users to do a major by major upgrade. That version
is the version that contains the index deletion problem fix and thus we
chose it as the minimum

**Comment by mikeminutillo** [comment]
That makes sense. I wonder if we should link to the latest 1.x, the latest 2.x and the latest release in the ServiceControl readme to make them easier to find

**Comment by boblangley** [comment]
The popup in scmu will also give them a link to the release page

**Pull request merged**